proxy/src/main/java/org/apache/rocketmq/proxy/grpc/OptionalSSLProtocolNegotiator.java

```diff
@@ -50,45 +60,83 @@ public AsciiString scheme() {
     }
 
     @Override
-    public ChannelHandler newHandler(GrpcHttp2ConnectionHandler grpcHttp2ConnectionHandler) {
-        ChannelHandler plaintext =
-            InternalProtocolNegotiators.serverPlaintext().newHandler(grpcHttp2ConnectionHandler);
-        ChannelHandler ssl =
-            InternalProtocolNegotiators.serverTls(sslContext).newHandler(grpcHttp2ConnectionHandler);
-        return new PortUnificationServerHandler(ssl, plaintext);
+    public ChannelHandler newHandler(GrpcHttp2ConnectionHandler grpcHandler) {
+        return new PortUnificationServerHandler(grpcHandler);
     }
 
     @Override
     public void close() {}
 
     public static class PortUnificationServerHandler extends ByteToMessageDecoder {
+
         private final ChannelHandler ssl;
         private final ChannelHandler plaintext;
 
-        public PortUnificationServerHandler(ChannelHandler ssl, ChannelHandler plaintext) {
-            this.ssl = ssl;
-            this.plaintext = plaintext;
+        public PortUnificationServerHandler(GrpcHttp2ConnectionHandler grpcHandler) {
+            this.ssl = InternalProtocolNegotiators.serverTls(loadSslContext())
```

Comment:
SslContext is only need to load once

Reply:
Yes, it has been fixed, please review again.